Recommendation Audiences GA [DOC-926] (#6787)
segmentio/segment-docs: src/engage/audiences/recommendation-audiences.md

---
title: Recommendation Audiences
plan: engage-foundations
---
Recommendation Audiences lets you select a parameter and then build an audience of the people that are most likely to engage with it. Segment optimized the personalized recommendations built by Recommendation Audiences for user-based commerce, media, and content affinity use cases. 

You can use Recommendation Audiences to power the following common marketing campaigns: 

- **Cross-selling**: Identify an audience of users who recently purchased a laptop and send those customers an email with a discount on items in the "laptop accessories" category. 
- **Upselling**: Identify an audience of users who regularly interact with your free service and send them a promotion for your premium service. 
- **Ranking**: Identify an audience of users who frequently interact with one category of your website and send them a promotion that contains only items from this category. 
- **Moving excess inventory**: Identify an audience of users who are in the top 5% of purchasers for a specific brand you sell and send them a coupon for the excess inventory you have of that brand. 

> - **Next best action**: Identify an audience of users who frequently read articles in your website's "sports" category and recommend those users your latest sports article. 

Author: ```suggestion
- **Next best action**: Identify an audience of users who frequently read articles in your website's "Sports" category and recommend those users your latest sports article. 
```

> - **Increasing average order value (AOV)**: Identify an audience of users who frequently interact with the "For Kids" section of your website and send them a back to school promotion in August, with free shipping after a set price threshold. 

Reviewer: In **Next best action**, "sports" category is lower case, but here, "For Kids" section is in upper case.  Was that intentional?  So like, types of categories are in lower case, but types of sections are upper case?

Author: Not intentional! I'll capitalize "Sports" to make it parallel

5. Click **Next**. 
6. Map event properties to the suggested model columns. <br> _(Optional)_: To add an additional column to your model, click **+ Add column** on the Map properties page. 
7. When you've completed your mappings, click **Save**. 

> warning ""
> Segment can take several hours to create your Recommendation Catalog. 

### Create your Recommendation Audience
Once you've created your Recommendation Catalog, you can build a Recommendation Audience. A Recommendation Audience lets you select a parameter and then build an audience of the people that are most likely to engage with that parameter. 

To create a Recommendation Audience: 
1. Open your Engage space and click **+ New audience**. 
2. Select **Recommendation Audience** and click **Next**. 
3. Select a property and value that you'd like to build your audience around (for example, if the property was "Company", you could select a value of "Twilio"). For values that haven't updated yet, enter an exact value into the **Enter value** field. If you're missing a property, return to your [Recommendation catalog](#set-up-your-recommendation-catalog) and update your mapping to include the property. 

> 4. Set an maximum audience size by selecting one of the pre-populated options, or move the slider to create a custom audience. Segment recommends audiences that contain less than the top 20% of your audience because as the size of your audience increases, the propensity to purchase typically decreases. See [Best practices](#best-practices) for more information.

Reviewer: ```suggestion
4. Set a maximum audience size by selecting one of the pre-populated options, or move the slider to create a custom audience. Segment recommends audiences that contain less than the top 20% of your audience because as the size of your audience increases, the propensity to purchase typically decreases. See [Best practices](#best-practices) for more information.
```

5. When you've filled out all fields, click **Next** to continue. 
6. On the Select Destinations page, select any destinations you'd like to sync your audience to and click **Next**.
7. Enter a name for your destination, update any optional fields, and click **Create Audience** to create your audience. 

> warning ""
> Segment can take up to a day to calculate your Recommendation Audience.

## Best practices

> - When mapping events to the model column during the set up process for your [Recommendation catalog](#set-up-your-recommendation-catalog), select the event property that matches the model column. For example, if you are mapping to model column ‘Brand’, select the property that refers to ‘Brand’ for each of the selected interaction events.

Reviewer: ```suggestion
- When mapping events to the model column during the setup process for your [Recommendation catalog](#set-up-your-recommendation-catalog), select the event property that matches the model column. For example, if you are mapping to model column ‘Brand’, select the property that refers to ‘Brand’ for each of the selected interaction events.
```

- Because a number of factors (like system load, backfills, or user bases) determine the complexity of an Audience, some compute times take longer than others. As a result, **Segment recommends waiting at least 24 hours for an Audience to finish computing** before you resume working with the Audience.

> - As the size of your audience increases, the propensity to purchase typically decreases. For example, an audience of 100k people that represents the top 5% of your customers might be more likely to purchase your product, but you might see a greater number of total sales if you expanded the audience to a million people that represent the top 50% of your customer base. 

Reviewer: ```suggestion
- As the size of your audience increases, the propensity to purchase typically decreases. For example, an audience of a hundred thousand people that represents the top 5% of your customers might be more likely to purchase your product, but you might see a greater number of total sales if you expanded the audience to a million people that represent the top 50% of your customer base. 
```

This is super minor, but I'd write out "a hundred thousand people" here, since we're writing it out later in the sentence ("a million people").  Or I'd change "a million" to a number.  Whatever is easiest to align them both.
